packages/manager/modules/sms/src/sms/sms/compose/translations/Messages_fr_FR.json

```diff
@@ -115,5 +115,7 @@
   "sms_sms_compose_add_phonebook_contact_empty": "Aucun contact trouvé.",
   "sms_sms_compose_add_phonebook_contact_adding": "Mise à jour des contacts…",
   "sms_sms_compose_add_phonebook_contact_added": "Liste des contacts mise à jour !",
-  "sms_sms_compose_add_phonebook_contact_ko": "Oups ! Impossible de récupérer la liste des contacts."
-}
\ No newline at end of file
+  "sms_sms_compose_add_phonebook_contact_ko": "Oups ! Impossible de récupérer la liste des contacts.",
+  "sms_sms_compose_message_commercial_description": "Pour des raisons légales, un SMS à caractère commercial ne sera envoyé qu’<strong>entre 8 h et 20 h, du lundi au samedi</strong>.",
```

Comment:
Is there a way to avoid `<strong>` tag in this sentence ? Maybe to split it ? Or having the whole sentence into `<strong>` tag ?

Reply:
Is this true for Spain too ?

Reply:
No there is no shortcode for Spain.

Reply:
Yet, we should probably avoid having opening hours directly in translations, an API depending on the user's susbsidiary would be better imo